telegram/_message.py

```diff
@@ -3588,6 +3591,9 @@ async def edit_text(
             of methods) or channel posts, if the bot is an admin in that channel. However, this
             behaviour is undocumented and might be changed by Telegram.
 
+        .. versionchanged:: NEXT.VERSION
+           New also passes :attr:`business_connection_id`.
```

Comment:
```suggestion
           Now also passes :attr:`business_connection_id`.
```
(same everywhere below)